src/data/set/basic.lean

```diff
@@ -1621,6 +1621,13 @@ begin
     convert congr_arg σ h; exact (σ.apply_symm_apply _).symm }
 end
 
+/-- The subtype of subsets of `t` is equivalent to `set ↥t`. -/
+def _root_.equiv.subset_equiv_set_coe (t : set α) : {s // s ⊆ t} ≃ set t :=
+{ to_fun := λ s, {x | ↑x ∈ s.1},
```

Comment:
```suggestion
{ to_fun := λ s, {x | ↑x ∈ (s : set α)},
```

Reply:
Why is this preferrable?

Reply:
Oh wait I get it, it's so that `simps` gives simp-normal lemmas.